2023/02/how-to-create-a-hugo-website-without-a-theme.md

```diff
@@ -0,0 +1,340 @@
+
+To keep it simple, my `style.scss` is less than 100 lines long. Here's part of it:
+
+```scss
+$paradise-pink: #EA1F4B;
+$paradise-pink-light: #EF5778;
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-size: 1.6rem;
+  line-height: 1.3;
+}
+
+h2, h3, h4, h5, h6 {
+  margin: 3.8rem 0 .6rem;
+}
+
+h1 {
+  font-size: 3.2rem;
+}
+
+h2 {
+  font-size: 2.5rem;
+}
+
+p, ul, ol {
+  font-size: 1.8rem;
+  margin: 0 0 .8em;
+}
+
+a {
+  text-decoration: none;
+  color: $paradise-pink;
+
+  &:hover {
+    color: $paradise-pink-light;
+  }
+}
+
+article {
+  padding: 2rem 1rem;
+  max-width: calc(44em + 2rem);
+  margin: 0 auto;
+}
+```
+
+> See [Aleksandr Hovhannisyan's article](https://www.aleksandrhovhannisyan.com/blog/62-5-percent-font-size-trick/) for a good explanation of setting the `html` font size to 62.5%.
+
+### Content
+
+As I mentioned earlier, Hugo's sections are defined by directories in the `content` directory. Here's what my notes look like after I organize them by university semester:
+
+```plain
+└── content
+    ├── _index.md
+    ├── winter-2022
+    │   ├── _index.md
+    │   ├── cs-235
+    │   │   ├── _index.md
+    │   │   ├── lecture-9.md
+    │   │   ├── lecture-10.md
+    │   │   └── quicksort-getting-started.md
+    └── winter-2023
+        ├── _index.md
+        └── ec-en-240
+            ├── _index.md
+            └── lecture-01-09.md
+```
+
+The [`_index.md`](https://gohugo.io/variables/page/#pages) files are used to add frontmatter and content to list pages. In this site, I just use them to name my sections. If you don't specify anything, it'll use the directory name.
+
+```plain
+---
+title: Electrical and Computer Engineering 240
+---
+```
+
+And here's one of those Markdown files; I'll use `lecture-9.md` as an example:
+
+```markdown
+---
+title: "Lecture 9"
+date: 2022-02-01T09:39:28-07:00
+katex: true
+---
+
+`sumto()` function: \\(\sum_1^i i\\)
+
+Can be done recursively: `sumto(5) = 5 + sumto(4)`
+
+Recursive function rules:
+
+1. have a base case (a case to stop the recursion)
+2. the recurive function must progress toward the base case (otherwise it will recurse infinitely)
+3. trust the induction
+4. Make sure that it won't make too many recursive calls
+5. Never run the same arguments of the recursive function twice. Caching may help
+```
+
+You may notice a `katex: true` key & value in the front matter. That's an optional extra goody I've added, which we'll go over soon!
+
+### Building and using the development server
+
+To deploy the site to the `build` directory, run:
+
+```plain
+$ hugo
+```
+
+Or to use Hugo's development server (one of its strongest features, in my opinion), go to the base site directory (`notes`) and run:
+
+```plain
+$ hugo serve
+```
+
+It will give you a localhost address with a port (`1313` unless it's taken) that you can open in your browser of choice.
```

Comment:
You say port 1313 will be used, but your example screenshots show 2 different ports other than this. I think it'd bring more harmony to the world to show port 1313 being used, to go along with what you said here.